tests/test_info_base.py

```diff
@@ -0,0 +1,268 @@
+"""Test alpenhorn.info_base"""
+
+import pytest
+import peewee as pw
+
+from alpenhorn.db import base_model
+from alpenhorn.info_base import (
+    info_base,
+    _NoInfo,
+    no_info,
+    acq_info_base,
+    file_info_base,
+    GenericAcqInfo,
+    GenericFileInfo,
+)
+
+
+def test_has_model():
+    """Test has_model"""
+
+    assert not info_base.has_model()
+    assert not _NoInfo.has_model()
+    assert acq_info_base.has_model()
+    assert file_info_base.has_model()
+    assert GenericAcqInfo.has_model()
+    assert GenericFileInfo.has_model()
+
+
+def test_is_acq():
+    """Test is_acq()"""
```

Comment:
Assuming this will be implemented in future PRs

Reply:
That's odd...

Reply:
OK.  I don't know what happened to it.  Presumably a casualty of one of the infinite rebases. I've re-implemented the test.